src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ChangeEvent } from 'react';
+
+import {
+  ActionGroup,
+  Alert,
+  AlertActionLink,
+  Button,
+  DropEvent,
+  FileUpload,
+  Modal,
+  ModalVariant,
+} from '@patternfly/react-core';
+
+interface ImportBlueprintModalProps {
+  setShowImportModal: React.Dispatch<React.SetStateAction<boolean>>;
+  isOpen: boolean;
+}
+
+export const ImportBlueprintModal: React.FunctionComponent<
+  ImportBlueprintModalProps
+> = ({ setShowImportModal, isOpen }: ImportBlueprintModalProps) => {
+  const onImportClose = () => {
+    setShowImportModal(false);
+  };
+  const [jsonContent, setJsonContent] = React.useState('');
+  const [filename, setFilename] = React.useState('');
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  const handleFileInputChange = (
+    _event: React.ChangeEvent<HTMLInputElement> | React.DragEvent<HTMLElement>,
+    file: File
+  ) => {
+    setFilename(file.name);
+  };
+  const handleClear = () => {
+    setFilename('');
+    setJsonContent('');
+  };
+
+  return (
+    <Modal
+      variant={ModalVariant.medium}
+      isOpen={isOpen}
+      title={'Import pipeline'}
+      onClose={onImportClose}
+    >
+      {false && (
+        <Alert
+          variant="info"
+          title="For expected file format, refer to documentation"
+          ouiaId="InfoAlert"
+        >
+          <AlertActionLink component="a" href="#">
+            View documentation
+          </AlertActionLink>
+        </Alert>
```

Comment:
`false` makes it never being render,  is this  for validation?
We can use `validated` prop and wrap the <FileUpload> with <FormGroup>

Reply:
We do not have the docs yet, there should be a link, but since we have no place to refer to, I wanted we might want to hide this for now.

Reply:
dead code looks weird and might accidently stay there forever, IMO its better just add the code once we have the docs and have a reminder in Jira instead of in code :)